core_java/grep/src/main/java/ca/jrvs/apps/grep/JavaGrep.java

```diff
@@ -0,0 +1,54 @@
+package ca.jrvs.apps.grep;
+
```

Comment:
git rm core_java/grep/src/main/java/ca/jrvs/apps/grep/JavaGrep.class

do not commit .class files